unlock-app/src/components/interface/GlobalErrorConsumer.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-
+import { connect } from 'react-redux'
 import { GlobalErrorContext } from '../../utils/GlobalErrorProvider'
 import Layout from './Layout'
 import { mapErrorToComponent } from '../creator/FatalError'
 
+// DEPRECATED we should not use the Context to detect errors
 const Consumer = GlobalErrorContext.Consumer
 
+// TODO: move that logic back to <GlobalErrorConsumer> once the Context logic is gone
```

Comment:
I would recommend we keep this behavior, as it allows changing the way errors are displayed. Although the paywall is splitting, we will add new apps that may desire to keep this feature. Your call, though, since it will soon be obsolete for the dashboard.